Source/JavaScriptCore/API/JSBigIntRef.h

```diff
@@ -0,0 +1,114 @@
+
+/*!
+    @function
+    @abstract         Creates a JavaScript bigint with a 64-bit signed integer.
+    @param ctx        The execution context to use.
+    @param integer    The 64-bit signed integer to copy into the new JSBigInt.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A JSBigInt representing the value of the integer, or NULL if an exception is thrown.
+*/
+JS_EXPORT JSBigIntRef JSBigIntCreateWithInt64(JSContextRef ctx, const int64_t integer, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Creates a JavaScript bigint with a 64-bit unsigned integer.
+    @param ctx        The execution context to use.
+    @param integer    The 64-bit unsigned integer to copy into the new JSBigInt.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A JSBigInt representing the value of the integer, or NULL if an exception is thrown.
+*/
+JS_EXPORT JSBigIntRef JSBigIntCreateWithUInt64(JSContextRef ctx, const uint64_t integer, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Creates a JavaScript bigint with an integer represented in string.
+    @param ctx        The execution context to use.
+    @param string     The JSString representation of an integer.
+    @param exception  A pointer to a JSValueRef in which to store an exception, if any. Pass NULL if you do not care to store an exception.
+    @result           A JSBigInt representing the value of the string, or NULL if an exception is thrown.
+*/
+JS_EXPORT JSBigIntRef JSBigIntCreateWithString(JSContextRef ctx, JSStringRef string, JSValueRef* exception) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Truncates a JavaScript bigint to a singed 64-bit integer and returns the resulting integer.
+    @param ctx        The execution context to use.
+    @param value      The JSBigInt to Truncate.
+    @result           A 64-bit signed integer representing the JSBigInt value.
+*/
+JS_EXPORT int64_t JSBigIntToBigInt64(JSContextRef ctx, JSBigIntRef value) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
+
+/*!
+    @function
+    @abstract         Truncates a JavaScript bigint to an unsigned 64-bit integer and returns the resulting integer.
+    @param ctx        The execution context to use.
+    @param value      The JSBigInt to Truncate.
+    @result           A 64-bit unsigned integer representing the JSBigInt value.
+*/
+JS_EXPORT uint64_t JSBigIntToBigUInt64(JSContextRef ctx, JSBigIntRef value) JSC_API_AVAILABLE(macos(JSC_MAC_TBA), ios(JSC_IOS_TBA));
```

Comment:
I think this should be `JSBigIntToUInt64` for symmetry with the inverse constructor.